rpcs3/util/cpu_stats.cpp

```diff
@@ -80,8 +80,16 @@ namespace utils
 		memcpy(&sys, &fsys, sizeof(FILETIME));
 		memcpy(&usr, &fusr, sizeof(FILETIME));
 		double percent = 1. * (sys.QuadPart - m_sys_cpu) + (usr.QuadPart - m_usr_cpu);
-		percent /= (now.QuadPart - m_last_cpu);
-		percent /= utils::get_thread_count();
+
+		if (const double cpu = (now.QuadPart - m_last_cpu))
+		{
+			percent /= cpu;
+		}
+
+		if (const u32 thread_count = utils::get_thread_count())
```

Comment:
This is never zero, atleast shoud be never be zero.

Reply:
Who knows

Reply:
I can improve the code. One sec

Reply:
I think non-windows code with division may have similar problem, although I never seen it.

Reply:
The non-windows code is already checking this.
I adjusted both codes paths to do the same.
The function now returns 0 in both cases.